CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+## How to contribute
+
+Fork the repository and make some changes.
+Once you're done with your changes send a pull request. You have to agree to
+the license. Thanks!
+If you have questions, visit us on irc.freenode.net in #opensuse-factory or
+ask on our mailing list opensuse-factory@opensuse.org
+
+### How to get started
+
+It can be beneficial to learn how openQA job is executed. To setup your own
+openQA instance, please refer to the documentation of [openQA project](https://github.com/os-autoinst/openQA).
+
+Tests developers guide can be found [here](https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc).
```

Comment:
*A* test developers guide.

I would be very careful with deep links. I guess L12 referencing the documentation is good enough. Or we mention http://open.qa

Reply:
what about this comment?

Reply:
Ok, now I've changed the line completely.